docs/docsite/rst/conf.py

```diff
@@ -25,6 +25,7 @@
 # sys.path.append(os.path.abspath('some/directory'))
 #
 sys.path.insert(0, os.path.join('ansible', 'lib'))
+sys.path.insert(0, os.path.abspath('.'))
```

Comment:
The idiomatic way is to inject the `_ext` dir so you could have multiple separate extensions loadable. Besides, no need to make things outside this directory importable.
https://github.com/ansible/awx-plugins/blob/0d569b5/docs/conf.py#L34C1-L34C48
```suggestion
sys.path.insert(0, os.path.abspath('_ext'))
```